src/cli.js

```diff
@@ -10,6 +10,17 @@ import path from "path"
 import resolveFrom from "resolve-from"
 import standalone from "./standalone"
 
+function resolveModule(moduleName: string): string {
+  // Should check these possibilities:
+  //   a. name of a node_module
+  //   b. absolute path
+  //   c. relative path relative to `process.cwd()`.
+  // If none of the above work, we'll try a relative path starting
+  // in `process.cwd()`.
+  return resolveFrom(process.cwd(), moduleName)
+    || path.join(process.cwd(), moduleName)
+}
+
```

Comment:
We should make `getModulePath()` from `augmentConfig.js` its own module, then use it both there and here.